frontends/web/src/routes/account/info/info.jsx

```diff
@@ -71,11 +71,15 @@ export default class Info extends Component {
             <div class="contentWithGuide">
                 <div class="container">
                     <Header title={<h2>{t('accountInfo.title')}</h2>} />
-                    <div class="innerContainer">
+                    <div class="innerContainer scrollableContainer">
                         <div class="content padded flex flex-column flex-center">
-                            <div class={[style.infoContent, 'box large'].join(' ')}>
-                                <SigningConfiguration info={info.signingConfiguration} code={code} />
-                            </div>
+                            {
+                                info.signingConfigurations.map((config, index) => (
+                                    <div key={index} class={[style.infoContent, 'box large'].join(' ')}>
```

Comment:
Better not use index as key, just to be save. Maybe use `config.keypath` as unique key?

haven't read it fully, just the first search result https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
what about no key at all? the list is not dynamic.